.github/workflows/zowe-cli-plugin.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```

Comment:
now that node14 is gone, we can remove any node14 specific steps from this workflow 😋